package.json

```diff
@@ -44,15 +45,17 @@
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "devDependencies": {
+    "@nrwl/angular": "^13.9.6",
     "@nrwl/cli": "13.9.6",
     "@nrwl/eslint-plugin-nx": "13.9.6",
     "@nrwl/express": "13.9.6",
     "@nrwl/jest": "13.9.6",
     "@nrwl/js": "13.9.6",
     "@nrwl/linter": "13.9.6",
     "@nrwl/node": "13.9.6",
-    "@nrwl/nx-cloud": "13.2.2",
+    "@nrwl/nx-cloud": "13.1.6",
     "@nrwl/react": "13.9.6",
+    "@nrwl/tao": "13.9.6",
```

Comment:
I believe this too was intended to be removed in the same PR as the `pre-commit` PR referenced earlier. Unless you have an explicit needs to keep this in, I'd leave it out.

Reply:
No this should have been removed. Fixed I think